### PR TITLE
PMM-9689 migrate from singlestat panel to stat

### PR DESCRIPTION
**Pull request by anton-bystrov**
[PMM-9689](https://jira.percona.com/browse/PMM-9689) Fix styles in all dashboards after upgrading to Grafana 8. In this PR was changed: upgrade panel types from singlestat to stat, fix font size.

Here before:
изображение
Here after:
изображение

### CODE REVIEW

**Comment by anton-bystrov** [comment]
> @anton-bystrov I see two issues related to the font settings on the Home dashboard: 1/ font size far too large, 32px would better look if reduced down to 24/26px 2/ all bold hurts the eyes, so better be normal os semi-bold

I'm changed font size to 24 px. And about bold type - we can't change it, because new type of panel haven't this settings.
изображение

**Review by tiagomotasantos — APPROVED** [review]

**Review by atymchuk — DISMISSED** [review]

File: dashboards/Insight/Advanced_Data_Exploration.json [hunk 1]
```diff
@@ -1,828 +1,830 @@
...
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": false,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 1,
+  "id": 29,
```
[the remark this answers is not in the document]
Comment (anton-bystrov) [reply dashboards/Insight/Advanced_Data_Exploration.json]: Done

File: dashboards/Experimental/Environment_Summary.json [hunk 2]
```diff
@@ -8,16 +8,23 @@
...
+          "type": "dashboard"
+        },
         "type": "dashboard"
       }
     ]
   },
   "description": "Detailed view on Environments",
   "editable": false,
+  "fiscalYearStartMonth": 0,
   "gnetId": 14372,
```
Comment (denisok) [inline dashboards/Experimental/Environment_Summary.json:24]: strange that this dashboard has `gnetId` field when others don't

**Review by denisok — APPROVED** [review]

File: dashboards/Experimental/Environment_Summary.json (hunk 2 above)
Comment (atymchuk) [reply dashboards/Experimental/Environment_Summary.json:24]: > strange that this dashboard has `gnetId` field when others don't

Not really, there is plenty of those, for example: https://github.com/percona/grafana-dashboards/pull/1319/files#diff-b187478b147047e0d77293f487ca4283b7c84b2434b79cf072d2554bdd5f56fbR54

So do we need it, won't it break future upgrades?

Comment (denisok) [reply dashboards/Experimental/Environment_Summary.json:24]: I would prefer to do it by book and set it to null.

**Review by denisok — CHANGES_REQUESTED** [review]
please set `gnetId` to null everywhere

File: dashboards/Experimental/Environment_Summary.json (hunk 2 above)
Comment (anton-bystrov) [reply dashboards/Experimental/Environment_Summary.json]: Done

**Review by denisok — DISMISSED** [review]

**Review by atymchuk — CHANGES_REQUESTED** [review]

**Review by denisok — APPROVED** [review]

**Review by denisok — APPROVED** [review]

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**